### PR TITLE
Feature/upgrading lumen 6 8

### PR DESCRIPTION
**Pull request by spaceo**
Upgrading Lumen and dependencies from 6 -> 7 -> 8.

(Includes the [local docker development setup](https://github.com/danskernesdigitalebibliotek/ddb-material-list/pull/12))

#### How to  review?
Go through commits.

### CODE REVIEW

File: app/Exceptions/Handler.php [hunk 1]
```diff
@@ -2,12 +2,12 @@
...
-use Exception;
-use Illuminate\Auth\Access\AuthorizationException;
-use Illuminate\Database\Eloquent\ModelNotFoundException;
-use Illuminate\Http\Exceptions\HttpResponseException;
+use Throwable;
 use Illuminate\Http\Response;
 use Illuminate\Validation\ValidationException;
+use Illuminate\Auth\Access\AuthorizationException;
+use Illuminate\Http\Exceptions\HttpResponseException;
+use Illuminate\Database\Eloquent\ModelNotFoundException;
```
Comment (kasperg) [inline app/Exceptions/Handler.php:10]: I wonder why these imports are moved around. It does not seem to be required. Previously they were ordered alphabetically.

File: composer.json [hunk 2]
```diff
@@ -54,7 +54,11 @@
     "config": {
         "preferred-install": "dist",
         "sort-packages": true,
-        "optimize-autoloader": true
+        "optimize-autoloader": true,
+        "allow-plugins": {
+            "phpstan/extension-installer": true,
+            "composer/package-versions-deprecated": true
+        }
```
Comment (kasperg) [inline composer.json:60]: This is needed for the Composer 2.2 update. Another time you might consider putting this in a separate commit.

File: bootstrap/app.php [hunk 3]
```diff
@@ -79,7 +80,7 @@
 */
 
 $app->register(\Adgangsplatformen\Support\Illuminate\AdgangsplatformenServiceProvider::class);
-$app->register(Spatie\Cors\CorsServiceProvider::class);
+$app->register(CorsServiceProvider::class);
```
Comment (kasperg) [inline bootstrap/app.php:84]: Is this something that they forgot to mention in the upgrade guide or did we miss something?

File: database/seeders/DatabaseSeeder.php [hunk 4]
```diff
@@ -1,5 +1,7 @@
 <?php
 
+namespace Database\Seeders;
+
 use Illuminate\Database\Seeder;
 
 // phpcs:ignore PSR1.Classes.ClassDeclaration.MissingNamespace
```
Comment (kasperg) [inline database/seeders/DatabaseSeeder.php:7]: :+1: With the namespace added we should be able to drop the `phpcs:ignore`.

**Review by kasperg — APPROVED** [review]
:+1: Changes in relation to the Lumen update looks good!

File: bootstrap/app.php (hunk 3 above)
Comment (spaceo) [reply bootstrap/app.php:84]: No it's from the Laravel upgrade guide: https://laravel.com/docs/7.x/upgrade#cors-support

**Pull request merged**